lib/order.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../lib/menu.rb'
+require 'date'
+require 'twilio-ruby'
+
+class Order < Menu
```

Comment:
Have a think about why you made `Order` inherit from `Menu`.  Is inheritance used for this purpose? Is the `Order` and `Menu` relationship an inheritance one?